drivers/docker/driver.go

```diff
@@ -1630,29 +1638,26 @@ func (d *Driver) DestroyTask(taskID string, force bool) error {
 		return err
 	}
 
-	c, err := dockerClient.InspectContainerWithOptions(docker.InspectContainerOptions{
-		ID: h.containerID,
-	})
+	c, err := dockerClient.ContainerInspect(d.ctx, h.containerID)
 	if err != nil {
-		switch err.(type) {
-		case *docker.NoSuchContainer:
+		if strings.Contains(err.Error(), NoSuchContainerError) {
```

Comment:
wouldn't `errors.Is` work here?

Reply:
is there a clever way to check against interface types like these?
https://pkg.go.dev/github.com/docker/docker@v27.1.1+incompatible/errdefs#ErrNotFound